apps/chat/src/store/import-export/importExport.epics.ts

```diff
@@ -143,13 +143,20 @@ const exportConversationsEpic: AppEpic = (action$, state$) =>
         new Set(conversationsListing.map((info) => info.folderId)),
       );
       //calculate all folders;
-      const folders = getFoldersFromPaths(
+      const foldersWithConversation = getFoldersFromPaths(
         Array.from(
           new Set(foldersIds.flatMap((id) => getAllPathsFromPath(id))),
         ),
         FolderType.Chat,
       );
 
+      const allFolders = ConversationsSelectors.selectFolders(state$.value);
+      const emptyFolders = allFolders.filter(
+        ({ id }) => !foldersIds.some((folderId) => folderId === id),
+      );
```

Comment:
this can be removed - `combineEntities` will merge by ID (ignore second duplicate)

Reply:
Thank you! 
Changed.